src/entities/StudentSettlement/model/services/initStudentSettlement/initStudentSettlement.ts

```diff
@@ -0,0 +1,40 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { AxiosError } from 'axios';
+import camelcaseKeys from 'camelcase-keys';
+
+import { ThunkConfig } from '@/app/providers/StoreProvider';
+import { Dorm } from '@/entities/Dorm';
+import { getBlocks } from '@/entities/Block';
```

Comment:
cross import throughout entities should be via @x folder.
For example import { Dorm } from '@/entities/Dorm/@x/StudentSettlement'